.gitignore

```diff
@@ -42,3 +42,5 @@ local.properties
 # Sandbox stuff
 _sandbox
 
+signing.properties
+droidconit.jks
```

Comment:
(this also means the signing.properties will be loads of fun -  just learn to live with that uncommitted file)